internal/encoding/asn1/rawvalue/rawvalue.go

```diff
@@ -0,0 +1,28 @@
+// Copyright The Notary Project Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rawvalue
+
+import (
+	"bytes"
+	"encoding/asn1"
+)
+
+// Equal returns true if two asn1.RawValue are equal
+func Equal(m asn1.RawValue, n asn1.RawValue) bool {
```

Comment:
We should have it put under `package internal/encoding/asn1` or maybe `package internal/encoding/asn1util` and have the function name `EqualRawValue`.

Reply:
Moved to `package internal/encoding/asn1`